bin/hhBtagTest.cc

```diff
@@ -0,0 +1,27 @@
+#include "../interface/HH_BTag.h"
+
+int main()
+{
+    hh_btag::HH_BTag test("/afs/cern.ch/work/m/mdidomen/hh-italian/Analysis_2018/CMSSW_10_2_16/src/hh-bbtautau/Studies/python/prova_pipo_v2_par1.pb");
```

Comment:
the weight file should be part of the repository and not some abs path in your home area.